src/Billable.php

```diff
@@ -69,6 +69,61 @@ public function refund($charge, array $options = [])
     }
 
     /**
+     * Determines if the customer currently has a card on file.
+     *
+     * @return bool
+     */
+    public function hasCardOnFile()
+    {
+        return (bool) $this->card_brand;
+    }
+
+    /**
+     * Synchronises the customer's card from Stripe back into our database.
+     *
+     * @return $this
+     */
+    public function syncCustomerCardDetails()
+    {
+        $customer = $this->asStripeCustomer();
+
+        // Let's go through all the users' sources (cards) until we find
+        // the default source for the customer. We then have to update
+        // it in our own database to determine if they have a card.
+        foreach ($customer->sources->data as $card) {
+            if ($card->id === $customer->default_source) {
+                $this->fillCardDetails($card)->save();
+
+                return $this;
+            }
+        }
+
+        // We no longer have a card? We'll remove it from our database.
+        $this->fillCardDetails(null)->save();
+
+        return $this;
+    }
+
+    /**
+     * Fills the user's properties with the source from Stripe.
+     *
+     * @param \Stripe\Card|null  $card
+     * @return $this
+     */
+    protected function fillCardDetails($card)
+    {
+        $this->card_brand = null;
```

Comment:
This looks like a possible change in behavior from the `updateCard` method, which did not `null` out the columns. Could be a breaking change of have unexpected behavior.